File: examples/node/address-resolve.mjs
import { OneDexClient } from '../../packages/js/src/index.js';

const client = new OneDexClient({
  baseUrl: process.env.ONEDEX_BASE_URL,
  apiKey: process.env.ONEDEX_API_KEY,
});

const response = await client.address.resolve('10 rue de la Paix, Paris');
console.log(JSON.stringify(response, null, 2));

File: examples/python/address_resolve.py
import json
import os
import sys
from pathlib import Path

sys.path.insert(0, str(Path(__file__).resolve().parents[2] / "packages/python/src"))

from onedex import OneDexClient


client = OneDexClient(
    base_url=os.environ.get("ONEDEX_BASE_URL"),
    api_key=os.environ.get("ONEDEX_API_KEY"),
)

response = client.address.resolve("10 rue de la Paix, Paris")
print(json.dumps(response, indent=2, ensure_ascii=False))
